scripts/test-in-lxd.sh

```diff
@@ -21,6 +21,20 @@ TESTER=subiquity-${IMAGE##*:}
 
 lxd init --auto
 
+# If the runner image uses docker, then work around a known connectivity issue
+# when using lxd and docker at the same time.
+# https://discuss.linuxcontainers.org/t/containers-do-not-have-outgoing-internet-access/10844/7
+# https://documentation.ubuntu.com/lxd/en/latest/howto/network_bridge_firewalld/#prevent-connectivity-issues-with-lxd-and-docker
+if [ "active" == "$(systemctl is-active docker)" ]
+then
+    iptables  -I DOCKER-USER -i lxdbr0 -j ACCEPT
+    iptables  -I DOCKER-USER -o lxdbr0 -m conntrack --ctstate RELATED,ESTABLISHED -j ACCEPT
+    # GitHub doesn't use IPV6 (yet?)
+    # ip6tables -I DOCKER-USER -i lxdbr0 -j ACCEPT
+    # ip6tables -I DOCKER-USER -o lxdbr0 -m conntrack --ctstate RELATED,ESTABLISHED -j ACCEPT
+fi
+
+
```

Comment:
Can we move this to a separate script that runs only in the CI please? `test-in-lxd.sh` can be used locally and this would:
1. force us to run the script as root if we have docker enabled
2. insert firewall rules that are not deleted when the script exits

Reply:
Yes, that's fine!